tests/unit/test_wheel.py

```diff
@@ -243,9 +245,8 @@ def test_unpack_wheel_no_flatten(self):
         from tempfile import mkdtemp
         from shutil import rmtree
 
-        filepath = '../data/packages/meta-1.0-py2.py3-none-any.whl'
-        if not os.path.exists(filepath):
-            pytest.skip("%s does not exist" % filepath)
+        filepath = os.path.join(DATA_DIR, 'packages',
```

Comment:
I thought about that, but using the data fixture result in a temporary copy of the data directory that is really not needed in this case.